### PR TITLE
added a function which uses "process.run"[v2]

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1- fix style compatible to pep8 standard
2-rename make to run_make which using process.run

### CODE REVIEW

File: avocado/utils/build.py [hunk 1]
```diff
@@ -10,17 +10,18 @@
 # See LICENSE for more details.
 #
 # Copyright: Red Hat Inc. 2013-2014
+#          : IBM 2016
 # Author: Lucas Meneghel Rodrigues <lmr@redhat.com>
+#       : Praveen K Pandey <praveen@linux.vnet.ibm.com>
```
Comment (ldoktor) [inline avocado/utils/build.py]: Well I wouldn't call this a big change, would you? (the purpose of authorship is to mark significant changes like new API, complete reworks etc. This change is only a minor modification which does not require the change of copyright)

Comment (PraveenPenguin) [inline avocado/utils/build.py]: @ldoktor sure i am also don't think this is big change can i remove this and rasie new PR ?

Comment (ldoktor) [inline avocado/utils/build.py]: See eg. the first response to http://programmers.stackexchange.com/questions/203014/author-has-inserted-copyright-into-code-with-gnu-public-license-notice-implica

Comment (ldoktor) [inline avocado/utils/build.py]: Force update is fine

**Comment by PraveenPenguin** [comment]
@ldoktor sure will do, remove this and rasie new PR but need you help regarding travis run i am not able to figure out 

https://travis-ci.org/avocado-framework/avocado/jobs/158379386
`ERROR: test_datadir_alias (selftests.functional.test_basic.RunnerOperationTest)`

File: avocado/utils/build.py [hunk 2]
```diff
@@ -58,9 +59,37 @@ def make(path, make='make', env=None, extra_args='', ignore_status=False,
...
+    make_process = process.run(cmd,
+                               env=env,
+                               ignore_status=ignore_status,
+                               allow_output_check=allow_output_check)
     os.chdir(cwd)
     return make_process
+
+
+def make(path, make='make', env=None, extra_arg='', ig_status=False,
+         allow_out_check='none'):
```
Comment (ldoktor) [inline avocado/utils/build.py]: Please don't change the API, which means don't change the argument names. This is what causes the CI failures and it's essential to keep them as they were before.

File: avocado/utils/build.py [hunk 3]
```diff
@@ -58,9 +59,37 @@ def make(path, make='make', env=None, extra_args='', ignore_status=False,
...
+                               to allow both standard output and error,
+                               and 'none', to allow none to be
+                               recorded (default). The default here is
+                               'none', because usually we don't want
+                               to use the compilation output as a reference
+                               in tests.
+    :type allow_out_check: str
+    """
+
+    status = run_make(path, make, env, extra_arg, ig_status, allow_out_check)
```
Comment (ldoktor) [inline avocado/utils/build.py]: This is not really status, it's `result`.

**Comment by ldoktor** [comment]
Hello @PraveenPenguin, the Travis failure is caused by the change in API you made. The arguments have to be preserved, not just the order. Additionally there is the `status` variable, which stores `result`, it'd be nice if you could address that. Last minor thing, it'd be nice to extend the two docstrings of `:returns: result of the make process`, resp `:returns: exit status of the make process`.

Apart from those it looks good.

**Comment by PraveenPenguin** [comment]
@ldoktor raised new PR #1456 please have a look .

**Pull request closed without merging**